src/Main.java

```diff
@@ -1,5 +1,68 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        //Task1
+        byte clientOS = 1;
+        if (0 == clientOS) {
+            System.out.println("Установите версию приложения для iOS по ссылке");
+        } else if (1 == clientOS) System.out.println("Установите версию приложения для Android по ссылке");
+        //Task2
+        //0 - iOS; 1 - Android
+        System.out.println("Task2");
+        byte os = 0;
+        short clientDeviceYear = 2018;
+        if (0 == os && clientDeviceYear < 2015) {
+            System.out.println("Установите ОБЛЕГЧЕННУЮ версию приложения для iOS по ссылке");
+        } else if (0 == os && clientDeviceYear >= 2015)
+            System.out.println("Установите версию приложения для iOS по ссылке");
+        if (1 == os && clientDeviceYear < 2015) {
+            System.out.println("Установите ОБЛЕГЧЕННУЮ версию приложения для Android по ссылке");
+        } else if (1 == os && clientDeviceYear >= 2015)
+            System.out.println("Установите версию приложения для Android по ссылке");
+        //Task3
+        int year = 1904;
+        if (year > 1584) {
+            if (year % 4 == 0 && year % 100 != 0 || year % 400 == 0) {
+                System.out.println("Високосный");
+            } else System.out.println("Невисокосный");
```

Comment:
код можно сократить до одного if/else:
```suggestion
        if (year > 1584 && year % 4 == 0 && year % 100 != 0 || year % 400 == 0) {
            System.out.println("Високосный");
        } else {
            System.out.println("Невисокосный");
        }
```